--- ../slurm-19.05.5.orig/src/slurmctld/proc_req.c	2019-12-20 20:44:51.000000000 +0000
+++ src/slurmctld/proc_req.c	2020-02-20 06:06:48.969789050 +0000
@@ -2747,16 +2747,16 @@
 		return;
 	}
 
-#if defined HAVE_FRONT_END
-	/* Limited job step support */
-	/* Non-super users not permitted to run job steps on front-end.
-	 * A single slurmd can not handle a heavy load. */
-	if (!validate_slurm_user(uid)) {
-		info("Attempt to execute job step by uid=%d", uid);
-		slurm_send_rc_msg(msg, ESLURM_NO_STEPS);
-		return;
-	}
-#endif
+//#if defined HAVE_FRONT_END
+//	/* Limited job step support */
+//	/* Non-super users not permitted to run job steps on front-end.
+//	 * A single slurmd can not handle a heavy load. */
+//	if (!validate_slurm_user(uid)) {
+//		info("Attempt to execute job step by uid=%d", uid);
+//		slurm_send_rc_msg(msg, ESLURM_NO_STEPS);
+//		return;
+//	}
+//#endif
 
 #if defined HAVE_NATIVE_CRAY
 	slurm_mutex_lock(&slurmctld_config.thread_count_lock);
